modules/ROOT/pages/metrics/configuring-dev-env-metrics.adoc

```diff
@@ -0,0 +1,14 @@
+include::{partialsdir}/attributes.adoc[]
+
+:service-name: Metrics
+
+= Configuring your Development Environment for the {service-name} Service
+
+* [x] You have provisioned one of the {product-name} as described in xref:metrics/provisioning.adoc[Provisioning {service-name} Service].
+* [x] You have associated the service with a {mobile-client} as described in xref:metrics/binding.adoc[Binding a {mobile-client} with the {service-name} Service].
```

Comment:
As mentioned i don't think Metrics _needs_ to be bound with a mobile client unless you want device metrics. Without any bindings metrics should already show data about the deployed services.